doc/whats_new/v0.20.rst

```diff
@@ -513,6 +513,12 @@ Classifiers and regressors
   per-tree basis. The previous behavior over-weighted the Gini importance of
   features that appear in later stages. This issue only affected feature
   importances. :issue:`11176` by :user:`Gil Forsyth <gforsyth>`.
+  
+- Fixed a bug in :class:`tree.MAE` to ensure sample_weights are being used 
```

Comment:
use backticks around sample_weights

Reply:
English wise, it didn't quite look right `sample_weight`s (with plurality) so changed to plain English (no parameter reference).